src/pymmcore_widgets/mda/_core_mda.py

```diff
@@ -88,11 +87,24 @@ def _on_sys_config_loaded(self) -> None:
     def value(self) -> MDASequence:
         """Set the current state of the widget."""
         val = super().value()
+
+        # if the z plan is relative, and there are no stage positions, add the current
```

Comment:
Can you add a comment here that this would ideally not be the final solution here.  Explaining that it would be better to put it on the runner somehow